signature/policy_config.go

```diff
@@ -33,6 +42,30 @@ func (err InvalidPolicyFormatError) Error() string {
 
 // FIXME: NewDefaultPolicy, from default file (or environment if trusted?)
 
+// DefaultPolicy returns the default policy of the system.
+// Most applications should be using this method to get the policy configured
+// by the system administrator.
+// ctx should usually be nil, can be set to override the default.
+// NOTE: When this function returns an error, either immediately abort,
+// or use the returned *Policy (which will never be nil); this policy
+// fails closed and refuses to trust any image.
+func DefaultPolicy(ctx *types.SystemContext) (*Policy, error) {
+	policyPath := defaultPolicyPath(ctx)
+	policy, err := NewPolicyFromFile(policyPath)
+	if err != nil {
+		policy = &Policy{Default: PolicyRequirements{NewPRReject()}}
+	}
+	return policy, err
```

Comment:
alright, got it but it's pretty confusing.

User of this function at https://github.com/projectatomic/skopeo/pull/157/files#diff-c6c318da062a6b9637745a480d10434eR88 bail out on error so why setting in line 56 a default policy if we're stil going to return the error here?